yarn.lock

```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
 
 
+"@cspotcode/source-map-support@^0.8.0":
```

Comment:
This file should not be checked in at all. Please add it to .gitignore so it doesn't get committed by accident again.

Reply:
Actually. I take it back. It may be that you want to keep the yarn.lock file, as it captures existing dependencies so one can detect changes to dependencies.  It gets regenerated, so keep in the tree if you see fit.

Reply:
@mbeckerle Yes I believe this was the reason we added it, was to ensure from developer to developer you make sure the dependencies are the same.